src/cmap/auth/mongodb_oidc/token_entry_cache.ts

```diff
@@ -0,0 +1,110 @@
+import type { OIDCMechanismServerStep1, OIDCRequestTokenResult } from '../mongodb_oidc';
+
+/* 5 minutes in milliseonds */
+const EXPIRATION_BUFFER = 300000;
+/* 5 hours in seconds */
+const DEFAULT_EXPIRATION = 18000;
```

Comment:
Are these values pulled directly from the spec? If not, can we consider standardizing our unit of time measurement? It would be nice if OIDC, all the timeouts and durations were in milliseconds. (or seconds)

Reply:
Yes the spec does define these so I kept them in the same units as the spec.

Reply:
If you think the units here might be confusing, I’d recommend to just incorporate them into the constant name (e.g. `EXPIRATION_BUFFER_MS`, `DEFAULT_EXPIRATION_SECS`, etc.)

Reply:
Updafted